### DISCUSSION TITLE
Creating foreign keys [Mar 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5388] [Discord Threads]
> 🙋‍♂️ **Jhon Mosk**

**Comment by o1lab** [comment]
> 🙋‍♂️ **kyleecodes**  
#### Sure, are you setting up foreign keys in the API or in the application?  
#  
> 🙋‍♂️ **Jhon Mosk**  
#### In app  
#  
> 🙋‍♂️ **kyleecodes**  
#### I think you are looking for a LinkToAnotherRecord column type: https://docs.nocodb.com/setup-and-usages/link-to-another-record/  
#  
> 🙋‍♂️ **Jhon Mosk**  
#### LinkToRecord is not that. In LinkToRecord, I have to manually link to a record in another table. And in the postgres foreign key, I once specify the dependency and all new records have it.  
#  
> 🙋‍♂️ **kyleecodes**  
#### You have a really good point. tagging <@&682859083799199814> for comment.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1075042048454959185)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
